samples/drivers/video/capture/src/main.c

```diff
@@ -125,10 +120,30 @@ int main(void)
 		return 0;
 	}
 
-	printk("- Default format: %c%c%c%c %ux%u\n", (char)fmt.pixelformat,
+	if (CONFIG_VIDEO_FORMAT_HEIGHT) {
+		fmt.height = CONFIG_VIDEO_FORMAT_HEIGHT;
+	}
+
+	if (CONFIG_VIDEO_FORMAT_WIDTH) {
+		fmt.width = CONFIG_VIDEO_FORMAT_WIDTH;
+		fmt.pitch = fmt.width * 2;
+	}
+
+	if (strcmp(CONFIG_VIDEO_PIXEL_FORMAT, "")) {
```

Comment:
I was about to suggest `strcmp(a, b) != 0` as it looks like `a != b`, but actually it is a tie!
```
$ grep -Er '\(strcmp\([^)]*\) [!=<>]+ 0\)' ~/zephyrproject/zephyr/*/ | wc -l
376
$ grep -Er '\(!?strcmp\([^)]*\)\)' ~/zephyrproject/zephyr/*/ | wc -l
324
```